src/server.rs

```diff
@@ -175,21 +188,13 @@ impl Server {
             }
         }
 
-        for (id, ref mut client) in self.connections.iter_mut() {
-            remove_player_on_disconnect!(client.fetch_bytes(), *id);
-
-            let mut player_input_x = 0.0;
-            let mut player_input_y = 0.0;
-            let mut player_shooting = false;
-            let mut player_activating_powerup = false;
+        for ref mut client in self.connections.iter_mut() {
```

Comment:
Is this `ref mut` required?

Reply:
Well I'm not the one who added it initially but I can try without

Reply:
It worked fine without so I have pushed a new commit